core/libs/editor/src/lib/editor.service.ts

```diff
@@ -229,22 +229,34 @@ export class EditorService {
   loadNewAspectModel(rdfAspectModel: string, isDefault?: boolean) {
     this.refreshSidebar();
     this.removeLastSavedRdf();
-    this.notificationsService.info('Loading model', null, null, 5000);
+    this.notificationsService.info({title: 'Loading model', timeout: 2000});
+
     return this.rdfService.loadModel(rdfAspectModel).pipe(
-      switchMap(rdfModel =>
-        this.validateCurrentModel(rdfModel).pipe(
-          tap(() => {
-            this.loadModel$.next(null);
-            localStorage.removeItem(ValidateStatus.validating);
-            if (!isDefault) {
-              this.notificationsService.info('Aspect Model loaded', null, null, 5000);
-            }
-          }),
-          switchMap(validationResOfLoadedRdfModel => this.loadExternalModels(rdfModel).pipe(map(() => validationResOfLoadedRdfModel))),
-          switchMap(validationResOfLoadedRdfModel => this.loadCurrentModel(rdfModel, rdfAspectModel, validationResOfLoadedRdfModel))
-        )
-      )
+      switchMap(loadedRdfModel =>
+        combineLatest([this.loadExternalModels(loadedRdfModel), this.loadCurrentModel(loadedRdfModel, rdfAspectModel)])
```

Comment:
This will emit twice since there are 2 observables in `combineLatest`. Which means the bellow message will be displayed 2 times in the right conditions
Please find another way to do this.